core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/decorator/submission/era/EraExperimentDecorator.java

```diff
@@ -69,7 +71,7 @@ public void buildSubmission() {
       Element studyRef = submission.createElementNS(null, "STUDY_REF");
       studyRef.setAttribute("refname", e.getStudy().getAlias());
       studyRef.setAttribute("refcenter", submissionProperties.getProperty("submission.centreName"));
-      if (e.getStudy().getAccession() != null && !"".equals(e.getStudy().getAccession())) {
+      if (e.getStudy().getAccession() != null && !isStringEmptyOrNull(e.getStudy().getAccession())) {
```

Comment:
this check could be condensed into 

    if (!isStringEmptyOrNull(e.getStudy().getAccession()))